### ISSUE TITLE
[Bug] Archive page shows image aspect ratio in ::after selector

### ISSUE THREAD

**Issue by worldoptimizer** [elementor/elementor#10949]
## Prerequisites

- [x ] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [ x] The issue still exists against the latest stable version of Elementor.

## Description

Using the archive widget in a template page shows the image aspect ratio in content after. Seams like somebody put it there in development and then it slipped through to deployment

## Steps to reproduce

Just use the widget with images that have an aspect ratio

<img width="1133" alt="Bildschirmfoto 2020-03-24 um 09 52 46" src="https://user-images.githubusercontent.com/1752114/77406989-c2a01280-6db5-11ea-934d-04decb66f2e9.png">

**Comment by princesshorsepower** [comment]
Same issue here:

```
.elementor-2919 .elementor-element.elementor-element-724ee848::after {
    content: "0.66";
}
```

**Comment by shilo-ey** [comment]
I merged this issue with #10874, which is on the same subject so we could deal with it more efficiently. 
Please feel free to continue the conversation there.

Thanks for your feedback!
